engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -248,11 +248,7 @@ private void preRenderUpdate(WorldRenderingStage renderingStage) {
         resetStats();
 
         currentRenderingStage = renderingStage;
-        if (currentRenderingStage == WorldRenderingStage.MONO || currentRenderingStage == WorldRenderingStage.LEFT_EYE) {
-            isFirstRenderingStageForCurrentFrame = true;
-        } else {
-            isFirstRenderingStageForCurrentFrame = false;
-        }
+        isFirstRenderingStageForCurrentFrame = currentRenderingStage == WorldRenderingStage.MONO || currentRenderingStage == WorldRenderingStage.LEFT_EYE;
```

Comment:
No. Please revert to previous code. For two reasons.
1. the new version is not as readable and this is a tricky part of the code (which one day I hope to handle very differently)
2. I'm working on this particular file and I will submit changes sometimes soon.